deploy/complete/helm-chart/provision/templates/secrets-global.yaml

```diff
@@ -2,7 +2,7 @@
 {{- $oadbWalletPw := .Values.global.oadbWalletPassword | default (randAlphaNum 12) }}
 {{- $oadbName := .Values.global.oadbName | default (printf "mushop%s" (randAlphaNum 12)) }}
 {{- $oadbUser := .Values.global.oadbUser | default (printf "user%s" (randAlphaNum 12)) }}
-{{- $oadbPw := .Values.global.oadbPassword | default (printf "%s" (randAlphaNum 12)) }}
+{{- $oadbPw := .Values.global.oadbPassword | default (printf "%s%s" (randAlpha 1) (randAlphaNum 12)) }}
```

Comment:
meets the restrictions, can even be more chars than 13:
```
Passwords can be from 1 to 30 characters.

The first character in an Oracle password must be a letter.

Only letters, numbers, and the symbols “#”, “_” and “$” are acceptable in a password.```

Reply:
Some cultures does not like the number 13, to avoid any references, like the santagate.

There's even religious references: https://en.wikipedia.org/wiki/13_(number)